flow_manager/src/modules/database/custom-jobs/custom-jobs.service.ts

```diff
@@ -38,13 +42,23 @@ export class CustomJobsService {
     return await this.customJobModel.findById(id);
   }
 
+  public async getPick<K extends keyof CustomJobsDocument>(
+    id: string,
+    projection: string[],
```

Comment:
Not sure, but I think we might be ble to use `(keyof CustomJobsDocument)[]` instead of `string[]` here?

Reply:
it compiles!